exercises/concept/go/.meta/Exemplar.elm

```diff
@@ -0,0 +1,53 @@
+module Go exposing (..)
+
+
+type Player
+    = Black
+    | White
+
+
+type alias Game =
+    { whiteCapturedStones : Int
+    , blackCapturedStones : Int
+    , player : Player
+    , error : String
+    }
+
+
+type alias Rule =
+    Game -> Result String Game
```

Comment:
This function type alias, is that something the student will be familiar with?

Reply:
Hi Erik, type alias is explained as part of records (to which it is tightly linked, which is a prerequisite of this xercise, so yes!